docs/conf.py

```diff
@@ -260,6 +260,7 @@
     nitpick_ignore.append((dtype, target))
 
 # Extra intersphinx in addition to what is already in sphinx-astropy
+intersphinx_mapping['astropy'] = ('https://docs.astropy.org/en/stable/', None)
```

Comment:
This should not be needed. It was included in `from sphinx_astropy.conf.v1 import *  # noqa` above.
```suggestion
```